.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -121,3 +121,22 @@ body:
       placeholder: v1.24.1
     validations:
       required: true
+
+  - type: input
+    id: postgresql_version
+    attributes:
+      label: PostgreSQL version
+      description: What version of PostgreSQL are you running? (see database -> image section in your `docker-compose.yml`)
```

Comment:
We could also point to the "Database information" Dashboard instead. (Top right).

Reply:
Perfect idea, will add it

Reply:
added